job-descriptions/software-engineer-distribution.md

```diff
@@ -0,0 +1,64 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Distribution
+
+## About us
+
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
+
+We value openness and transparency. [Our handbook](https://about.sourcegraph.com/handbook), [our strategy](https://about.sourcegraph.com/company/strategy), and [our code](https://github.com/sourcegraph/sourcegraph) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+## About the role
+
+As a member of the [distribution team](https://about.sourcegraph.com/handbook/engineering/distribution) at Sourcegraph, you will focus on making Sourcegraph easy to build, deploy, configure, and monitor, on customer instances and on Sourcegraph.com (the largest Sourcegraph installation in the world). You will have the opportunity to directly interact with our customers to understand the problems that they face and iteratively design solutions to those problems. Read our [project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.mi8zg2ql2uc6) to see the challenges that we are working on today and in the near the future.
+
+### Qualifications
+
+We are looking for candidates with the following skills and a strong desire to learn:
```

Comment:
```suggestion
We are looking for candidates with the following skills:
```

(this language has proven confusing to candidates about whether the following skills are required or not)